inductiva/templates/heat_sink/openfoam/files/constant/fluid/fvOptions

```diff
@@ -0,0 +1,32 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+| =========                 |                                                 |
+| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
+|  \\    /   O peration     | Version:  v1912                                 |
+|   \\  /    A nd           | Website:  www.openfoam.com                      |
+|    \\/     M anipulation  |                                                 |
+\*---------------------------------------------------------------------------*/
+FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    location    "constant";
+    object      fvOptions;
+}
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+temperatureLimit
+{
+    type limitTemperature;
+    active true;
+
+    limitTemperatureCoeffs
+    {
+        selectionMode all;
+        min 270;
```

Comment:
I think they are just some limits to avoid divergences in the solver. For reasonable initial temperatures and heater powers, they shouldn't play a big role in the simulations.